File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_api_result_page_plus_game_growth_record_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultPagePlusGameGrowthRecordVO:
    """API调用结果"""
    data: PagePlusGameGrowthRecordVO
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str

File: sdkwork-backend-sdk-typescript/src/types/plus-api-result-page-plus-game-growth-record-vo.ts
import type { PagePlusGameGrowthRecordVO } from './page-plus-game-growth-record-vo';

/** APIè°ç¨ç»æ */
export interface PlusApiResultPagePlusGameGrowthRecordVO {
  /** Response data */
  data: PagePlusGameGrowthRecordVO;
  /** Response code: 2000=success, 4xxx=business failure, 5xxx=server error */
  code: string;
  /** Business message */
  msg: string;
  /** Request identifier */
  requestId: string;
  /** Client IP address */
  ip?: string;
  /** Server hostname */
  hostname?: string;
  /** Business error name */
  errorName: string;
}
